test/plugins/inlineStyles.13.svg

```diff
@@ -12,9 +12,7 @@
             /* Atrules with block */
             @font-face {
                 font-family: SomeFont;
-                src: local("Some Font"),
-                local("SomeFont"),
-                url(SomeFont.ttf);
+                src: local("Some Font"), local("SomeFont"), url(SomeFont.ttf);
```

Comment:
`csso` preserves the newlines in this `src` value, which were previously being turned into spaces by the `sax` normalization.